adapters/inmobi/inmobi.go

```diff
@@ -0,0 +1,97 @@
+package inmobi
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type InmobiAdapter struct {
+	endPoint string
+}
+
+func NewInmobiAdapter(endpoint string) *InmobiAdapter {
+	return &InmobiAdapter{
+		endPoint: endpoint,
+	}
+}
+
+func (a *InmobiAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+
+	for _, imp := range request.Imp {
```

Comment:
I expect this to break once you address my comment below, since `imp` is actually a shallow copy and this is only working today because you're not making a copy of the `imp.Banner` reference. You should use a standard index based for loop here and then reference the imp by index when setting the new banner. Like `for i := range request.Imp`.